.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: taiki-e/create-gh-release-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Please use the `release-a-changelog` action instead here, so that the release will get the text from the patch notes as well:
```yaml
    - uses: rasmus-saks/release-a-changelog-action@v1.0.1
      with:
        github-token: '${{ secrets.GITHUB_TOKEN }}'
```

Reply:
Thanks for the review. Updated!